packages/react/src/components/Footer/LocaleButton.js

```diff
@@ -1,7 +1,5 @@
-import {
-  featureFlag,
-  FOOTER_LOCALE_BUTTON,
-} from '../../internal/FeatureFlags.js';
+import { FOOTER_LOCALE_BUTTON } from '../../internal/FeatureFlags.js';
+import { featureFlag } from '@carbon/ibmdotcom-utilities';
```

Comment:
@photodow can you publish an alpha version of this from your local then reference that in package dependencies? 
cc: @annawen1

Reply:
looks like it's done. thanks!